catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/mappings/CswRecordMapperFilterVisitor.java

```diff
@@ -176,10 +187,19 @@ public Object visit(PropertyIsEqualTo filter, Object extraData) {
             sourceIds.add((String) ((Literal) filter.getExpression2()).getValue());
             return null;
         }
+        AttributeType type =
+                attributeTypes.get(((PropertyName) filter.getExpression1()).getPropertyName());
+        if (type != null) {
+            type.getBinding()
```

Comment:
Should you be doing something with the results of this call? Are there side-effects you're depending on instead? If so, that should probably be commented.